docs/build_conf.md

```diff
@@ -25,17 +26,17 @@ The default configuration can be overridden by using more options in `cmake`.<br
 For example, to build the `STM32F429ZI` device:
 - BUILD: Debug mode
 - DA: ECDSA-256
-- AES_MODE: GCM
+- AES_MODE: CCM
```

Comment:
Why are we changing GCM to CCM?

Reply:
it's an example for a custom build is we want to change build options